src/components/search/styles.css

```diff
@@ -42,14 +54,24 @@
   display: grid;
   z-index: 2;
   -webkit-overflow-scrolling: touch;
-  position: absolute;
   left: 0;
   top: 3em;
-  width: 90%;
   overflow: hidden;
   border: 1px solid #979797;
   background: white;
   border-radius: 4px;
+  position: absolute;
+  width: 90%;
+}
+
+
+@media (max-width: 768px) {
+  .hits-wrapper {
+    position: fixed;
+    width: 100%;
+    right: 0;
+    left: 0;
+  }
```

Comment:
we might want to use JS to have a mobile prop for the search box

https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia

eg

```
const isMobile = window.matchMedia('(max-width: 768px)');
...
<Hits isMobile={isMobile} 
```

NOTE, this might be a bad idea, but i'm trying to keep the styling out of styles.css

Reply:
I think it is a good idea, we should use something like this with react hooks to handle window resising

Reply:
nice idea